src/examples/mfem_common_plane.cpp

```diff
@@ -67,11 +67,12 @@ int main( int argc, char** argv )
   axom::slic::setIsRoot( rank == 0 );
 
   // define command line options
-  // number of times to uniformly refine the serial mesh before constructing the
-  // parallel mesh
+  // number of times to uniformly refine the serial mesh before constructing the parallel mesh
   int ref_levels = 2;
   // polynomial order of the finite element discretization
   int order = 1;
```

Comment:
Consider a clarifying comment if order has to be 1, or if a user/developer could have order > 1 if the mesh builder can support this.